README.md

```diff
@@ -164,11 +164,118 @@ outcome of an authorisation attempt using different configurations of
 user/record pairs based on application state. These variations should be
 organised into separate RSpec context blocks, as in the previous example.
 
+## Testing For an Allow List of Permitted Actions
+
+As of Pundit Matchers 2.1, the recommended approach is to use an explicit
+allow list of actions permitted by the policy (`permit_only_actions`) along with
+an explicit deny list of actions that are forbidden (`forbid_only_actions`).
```

Comment:
After another read, it looks like that the text is advising to use `permit_actions` and `forbid_actions`, but the code uses their `only` versions. I understand the need to be explicit at least once, but it is what `permit_only` is trying to avoid, or it maybe needs a clarification on why this is tested twice

Reply:
I've split them into two code examples. Hopefully it's worded closer to your original intent now?